mzLib/mzPlot/mzPlot.csproj

```diff
@@ -1,11 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net6.0-windows</TargetFramework>
+    <TargetFramework>net8.0-windows7.0</TargetFramework>
```

Comment:
Does it work to delete the `7.0` here?

Reply:
I believe we need the windows7.0 here to display the plots in WPF through MetaDraw